zoo-css-challenge/style.css

```diff
@@ -29,8 +29,12 @@ section {
   background-color: #f8f0fb;
 }
 
+.introduction {
+  background-color: white;
```

Comment:
There's an existing `.introduction` selector below on line 82 so we could put `background-color: white;` there instead